From f7d10b0e14c6db77ca1f0d3df43aa6d9372fffcf Mon Sep 17 00:00:00 2001
From: "khoa.dang" <khoa.dang@cavliwireless.com>
Date: Tue, 23 Apr 2024 21:52:46 +0700
Subject: [PATCH] qssi::build:: include target for spi demo


diff --git a/qssi/qssi.mk b/qssi/qssi.mk
index 6bdc176..a3abf0d 100644
--- a/qssi/qssi.mk
+++ b/qssi/qssi.mk
@@ -319,9 +319,13 @@ PRODUCT_PACKAGES += \
 
 PRODUCT_PACKAGES += MqttListener
 
+# -include $(TOPDIR)vendor/cavli/common/config/device-vendor.mk
+
 ###################################################################################
 # This is the End of target.mk file.
 # Now, Pickup other split product.mk files:
 ###################################################################################
 $(call inherit-product-if-exists, vendor/qcom/defs/product-defs/system/*.mk)
 ###################################################################################
+
+include vendor/common/common/config/qssi_spi.mk
-- 
2.40.1

