From 3a2ed7143ee3ab2b4ec6c06977f7e448c07f37c4 Mon Sep 17 00:00:00 2001
From: Trinh Viet Quang <trinh.quang@cavliwireless.com>
Date: Thu, 25 Apr 2024 16:47:25 +0700
Subject: [PATCH] device_tree:: Update device tree add UART


diff --git a/qcom/proprietary/devicetree-4.19/qcom/scuba-iot-idp.dtsi b/qcom/proprietary/devicetree-4.19/qcom/scuba-iot-idp.dtsi
index 945b30989..a927394f1 100755
--- a/qcom/proprietary/devicetree-4.19/qcom/scuba-iot-idp.dtsi
+++ b/qcom/proprietary/devicetree-4.19/qcom/scuba-iot-idp.dtsi
@@ -525,7 +525,7 @@
 };
 
 &qupv3_se5_spi {
-	status = "ok";
+	status = "disabled";
 	// can-controller@0 {
 	// 	compatible = "qcom,nxp,s32k144";
 	// 	reg = <0>;
diff --git a/qcom/proprietary/devicetree-4.19/qcom/scuba-pinctrl.dtsi b/qcom/proprietary/devicetree-4.19/qcom/scuba-pinctrl.dtsi
index bb2225abe..db10f977d 100755
--- a/qcom/proprietary/devicetree-4.19/qcom/scuba-pinctrl.dtsi
+++ b/qcom/proprietary/devicetree-4.19/qcom/scuba-pinctrl.dtsi
@@ -81,6 +81,33 @@
 				};
 			};
 		};
+		qupv3_se5_2uart_pins: qupv3_se5_2uart_pins {
+			qupv3_se5_2uart_active: qupv3_se5_2uart_active {
+				mux {
+					pins = "gpio16", "gpio17";
+					function = "qup5";
+				};
+
+				config {
+					pins = "gpio16", "gpio17";
+					drive-strength = <2>;
+					bias-disable;
+				};
+			};
+
+			qupv3_se5_2uart_sleep: qupv3_se5_2uart_sleep {
+				mux {
+					pins = "gpio16", "gpio17";
+					function = "gpio";
+				};
+
+				config {
+					pins = "gpio16", "gpio17";
+					drive-strength = <2>;
+					bias-pull-down;
+				};
+			};
+		};
 		/* SDC pin type */
 		sdc1_clk_on: sdc1_clk_on {
 			config {
diff --git a/qcom/proprietary/devicetree-4.19/qcom/scuba-qupv3.dtsi b/qcom/proprietary/devicetree-4.19/qcom/scuba-qupv3.dtsi
index a12d1e17b..3b2c75c8a 100755
--- a/qcom/proprietary/devicetree-4.19/qcom/scuba-qupv3.dtsi
+++ b/qcom/proprietary/devicetree-4.19/qcom/scuba-qupv3.dtsi
@@ -66,6 +66,22 @@
 		status = "disabled";
 	};
 
+	qupv3_se5_2uart: qcom,qup_uart@4a94000 {
+		compatible = "qcom,msm-geni-console";
+		reg = <0x4a94000 0x4000>;
+		reg-names = "se_phys";
+		interrupts = <GIC_SPI 332 IRQ_TYPE_LEVEL_HIGH>;
+		clock-names = "se-clk", "m-ahb", "s-ahb";
+		clocks = <&gcc GCC_QUPV3_WRAP0_S5_CLK>,
+			<&gcc GCC_QUPV3_WRAP_0_M_AHB_CLK>,
+			<&gcc GCC_QUPV3_WRAP_0_S_AHB_CLK>;
+		pinctrl-names = "default", "sleep";
+		pinctrl-0 = <&qupv3_se5_2uart_active>;
+		pinctrl-1 = <&qupv3_se5_2uart_sleep>;
+		qcom,wrapper-core = <&qupv3_0>;
+		status = "disabled";
+	};
+
 	qupv3_se0_i2c: i2c@4a80000 {
 		compatible = "qcom,i2c-geni";
 		reg = <0x4a80000 0x4000>;
diff --git a/qcom/proprietary/devicetree-4.19/qcom/scuba.dtsi b/qcom/proprietary/devicetree-4.19/qcom/scuba.dtsi
index ae190735e..b80899e92 100755
--- a/qcom/proprietary/devicetree-4.19/qcom/scuba.dtsi
+++ b/qcom/proprietary/devicetree-4.19/qcom/scuba.dtsi
@@ -42,6 +42,8 @@
 		sdhc2 = &sdhc_2; /* SDC2 SD Card slot */
 		swr0 = &swr0;
 		swr1 = &swr1;
+		serial0 = &qupv3_se4_2uart;
+		serial1 = &qupv3_se5_2uart;
 	};
 
 	cpus {
@@ -2703,6 +2705,10 @@
 	status = "ok";
 };
 
+&qupv3_se5_2uart {
+	status = "ok";
+};
+
 &qupv3_se3_4uart {
 	status = "ok";
 };
-- 
2.40.1

