From 5c3a63248e765fc8e449913fb5f792b9a88b8beb Mon Sep 17 00:00:00 2001
From: "khoa.dang" <khoa.dang@cavliwireless.com>
Date: Mon, 5 Aug 2024 09:36:00 +0000
Subject: [PATCH] add


diff --git a/bengal_2w/bengal_2w.mk b/bengal_2w/bengal_2w.mk
index 2986b42..14b4023 100644
--- a/bengal_2w/bengal_2w.mk
+++ b/bengal_2w/bengal_2w.mk
@@ -406,3 +406,6 @@ $(call inherit-product-if-exists, vendor/qcom/defs/product-defs/vendor/*.mk)
 
 # Add the vendor specific product.mk files
 $(call inherit-product-if-exists, vendor/cavli/cqs290/cqs290.mk)
+
+include vendor/common/common/config/cavlidemo.mk
+
-- 
2.7.4

