From c3d0318617c9bf56d8a800bdcef757ad4311020a Mon Sep 17 00:00:00 2001
From: "khoa.dang" <khoa.dang@cavliwireless.com>
Date: Wed, 8 May 2024 04:41:21 +0000
Subject: nesh::demo:: Include for demo


diff --git a/bengal_2w/bengal_2w.mk b/bengal_2w/bengal_2w.mk
index 02729e2..5d05900 100644
--- a/bengal_2w/bengal_2w.mk
+++ b/bengal_2w/bengal_2w.mk
@@ -399,3 +399,5 @@ $(call inherit-product-if-exists, vendor/qcom/defs/product-defs/system/*.mk)
 $(call inherit-product-if-exists, vendor/qcom/defs/product-defs/vendor/*.mk)
 ###################################################################################
 include vendor/cavli/cqs290/cqs290.mk
+
+include vendor/common/common/config/cavlidemo.mk
-- 
2.7.4

